seatunnel-translation/seatunnel-translation-flink/seatunnel-translation-flink-common/src/main/java/org/apache/seatunnel/translation/flink/source/FlinkRowCollector.java

```diff
@@ -42,9 +40,7 @@
 @Slf4j
 public class FlinkRowCollector implements Collector<SeaTunnelRow> {
 
-    private ReaderOutput<Row> readerOutput;
-
-    private final FlinkRowConverter rowSerialization;
```

Comment:
Do we need to remove the FlinkRowConverter Class itself?

Reply:
Not need, because the relevant metrics statistics and current limiting logic are all executed within this.

Reply:
I'm sorry, is my expression has a problem, I mean the org. Apache. Seatunnel. Translation. Flink. Serialization. FlinkRowConverter responsible for conversion of classes

Reply:
Get your means, done, please review again.